### ISSUE TITLE
Accessibility issue: Heading levels should only increase by one on /course-view

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#269]
## What
An accessibility scan flagged the element `<h5 class="card-title text-midnight mt-1 hover-text-underline"><span class="field field--name-title field--type-string field--label-hidden">GIST 330 Introduction to Remote Sensing</span>
</h5>` because heading levels should only increase by one. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/heading-order?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/dc8330ff-fd5d-4935-a216-c7e3f8d7dc52.png)

To fix this, ensure the order of headings is semantically correct.

Specifically:

Fix any of the following:
-   Heading order invalid

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
